controlled-vocabulary/src/test/java/org/uniprot/cv/evidence/GoEvidencesPropertyLocationTest.java

```diff
@@ -0,0 +1,68 @@
+package org.uniprot.cv.evidence;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.uniprot.cv.common.CVSystemProperties.GAF_ECO_LOCATION;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Optional;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.uniprot.cv.Reloader;
+
+@TestInstance(Lifecycle.PER_CLASS)
+class GoEvidencesPropertyLocationTest {
```

Comment:
Should all `Go...` names -> `GO...`? It stands for Gene Ontology, so should be GO like we did in QuickGO?

(sorry this comment is a bit off topic!)